packages/web-components/fast-components/src/accordion-item/accordion-item.styles.ts

```diff
@@ -110,6 +110,7 @@ export const AccordionItemStyles = css`
     .start {
         display: flex;
         align-items: center;
+        padding-left: calc(var(--design-unit) * 1px);
```

Comment:
@litteredwitherrors would this always be left, or should this us a logical property (`padding-inline-start`?) to ensure it works in RTL as well?

Reply:
Ah, yeah, that's probably better.